### ISSUE TITLE
Cucumber 2.0 compatibility

### ISSUE THREAD

**Issue by dbrock** [guard/guard-cucumber#24]
Cucumber 2.0 is about to be released, and it has some incompatible changes that break `guard-cucumber`  --- see e.g. <https://travis-ci.org/guard/guard-cucumber/jobs/56109933>.

As a temporary fix, I'm restricting the dependency in `guard-cucumber.gemspec` from `>= 1.3.0` to `~> 1.3` as part of the (unrelated) changes in <https://github.com/guard/guard-cucumber/pull/21>.

**Comment by e2** [comment]
@dbrock - thanks for heads up!

Could you create a separate pull request just for the fix in the gemspec?

It should be separate, because we'd had to release guard-cucumber 1.5.5 with this first.

Then, compatibility with cucumber 2.0 will probably have to be in guard-cucumber 1.6.0 (I doubt we'll need guard-cucumber 2.0 as well, but I'm not suer) - and that should be done before any other work.

**Comment by e2** [comment]
Fixed in 1.6.0, which uses Cucumber 2.x.

Thanks!
